internal/pkg/aws/cloudformation/stackset/status.go

```diff
@@ -28,6 +31,16 @@ func (s OpStatus) IsCompleted() bool {
 	return s.IsSuccessful() || s.IsFailure()
 }
 
+// InProgress returns true if the operation is not in a final state yet.
```

Comment:
Is it true? How about not started or failed.
```suggestion
// InProgress returns true if the operation is in progress.
```
Maybe something like that

Reply:
Updated!